client/my-sites/plugins/plugin-site/plugin-site.jsx

```diff
@@ -19,10 +18,6 @@ module.exports = React.createClass( {
 			return;
 		}
 
-		if ( ! this.props.wporg ) {
```

Comment:
This one is a bit confusing though.
Since `PluginSite` is used at `/plugins/:pluginName` to list the sites a plugin can be install on, I think it makes sense to disable the "Available sites" list component for non wporg plugins (in plugins.jsx)
Otherwise we'll get a nice error if we click on Install:
![image](https://cloud.githubusercontent.com/assets/230230/13029538/7f8989b0-d28f-11e5-9087-96c3e38cf30a.png)